state_machine/applications/flight/lib/mekf.py

```diff
@@ -11,16 +11,17 @@
     - The gyro bias (β) is estimated.
 """
 try:
-    from ulab.numpy import dot as matmul, eye as I, zeros, array, linalg  # noqa: E741 (I is not ambiguous)
+    from ulab.numpy import dot as matmul, eye as I, zeros, array, linalg, concatenate as concat  # noqa: E741
 except Exception:
-    from numpy import linalg, matmul, eye as I, zeros, array  # noqa: E741 (I is not ambiguous)
+    from numpy import linalg, matmul, eye as I, zeros, array, concatenate as concat  # noqa: E741
 from lib.mathutils import quaternion_mul, quaternion_to_left_matrix, hat, block, quaternion_to_rotation_matrix
 from math import cos, sin
 
-q = array([[0], [0], [0], [0]])  # Quaternion attitude vector
-β = array([[0], [0], [0]])  # Gyro bias vector
+q = array([0, 0, 0, 0])  # Quaternion attitude vector
```

Comment:
it would be a good idea to initialize with floats rather than ints, otherwise a type conversion has to be done at some point. 

`q = array([0., 0., 0., 0.])`

Reply:
Done